src/modules/feed/components/Transaction/Header.js

```diff
@@ -3,7 +3,7 @@ import { Dropdown } from "semantic-ui-react";
 import moment from "moment-twitter";
 import { route } from "preact-router";
 
-import { PATH_ADD_TRANSACTION } from "../../../../../routes";
+import { PATH_ADD_TRANSACTION } from "../../../../routes";
```

Comment:
For readability, would it help to link to it from the root? (using a webpack resolve alias?)